operator/internal/controller/usersignup_controller.go

```diff
@@ -51,79 +50,94 @@ type UserSignupReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.14.1/pkg/reconcile
 func (r *UserSignupReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	l := log.FromContext(ctx)
+	l := log.FromContext(ctx).WithValues("request", req)
 
 	u := toolchainv1alpha1.UserSignup{}
 	if err := r.Client.Get(ctx, req.NamespacedName, &u); err != nil {
 		if kerrors.IsNotFound(err) {
-			err := r.ensureWorkspaceIsDeleted(ctx, req.Name)
-			if errors.Is(err, ErrNonTransient) {
-				l.Error(err, "can not delete workspace", "user", req.Name)
-				return ctrl.Result{}, nil
+			l.V(6).Info("UserSignup not found")
```

Comment:
hmmm, why the explicit log levels here and later in this method?  If you can't use one of the other methods (i.e. Info or Error), please leave a comment.

Reply:
As per my understanding of how this logger works, you can set the log verbosity level via `V(int)` (higher int value means higher verbosity), otherwise it will use a default value. `Info` takes only a message, `Error` takes an error and a message.
To log a Debug line you will use the `V(int)` to set the log level that's Debug for you and then call `Info`.

In a subsequent PR, we may consider to add a package with a set of constants for log levels so we do use constants instead of numbers.